### ISSUE TITLE
Error in min(c(x, xmin), na.rm = TRUE) : invalid 'type' (list) of argument

### ISSUE THREAD

**Issue by ekernf01** [tengfei/ggbio#111]
Hi ggbio devs,

Have you solved https://github.com/tidyverse/ggplot2/issues/2610 yet? In case you haven't, I might fork this repo and look into it. I ran into the error using the following versions of ggbio and ggplot. 

Thanks!

```
 ggbio                               * 1.26.1     2018-03-09 Bioconductor                   
 ggplot2                             * 3.0.0      2018-07-03 CRAN (R 3.4.4)            
```

**Comment by lawremi** [comment]
Not sure, please check, and if not please feel free to submit a PR.

**Comment by ekernf01** [comment]
I'm sorry, I looked into this, but I don't know anything about ggplot's internals, and I ended up in way over my head.

**Comment by lawremi** [comment]
I'm pretty sure the error has been fixed, at least in Bioc 3.8.
